github-connector/internal/application_registry/sendregisterrequest.go

```diff
@@ -0,0 +1,97 @@
+package registerservice
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/model"
+)
+
+//RegisterResponse contain structure of response json
+type RegisterResponse struct {
+	ID string
+}
+
+//RegisterConfig contain configs
+type RegisterConfig struct {
+	HTTPClient  *http.Client
+	HTTPRequest *http.Request
+}
+
+//RequestConfig contain configs to create http requests
+type RequestConfig struct {
+	Type string
+	URL  string
+	Body io.Reader
+}
+
+//CreateJSONRequest - create http request, add headers and return client
+func CreateJSONRequest(config RequestConfig) (*http.Request, error) {
+	req, err := http.NewRequest(config.Type, config.URL, config.Body)
+
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+
+	return req, nil
+}
+
+//SendJSONRequest - create json struct and try post it into application-register's url
+func SendJSONRequest(config RegisterConfig) (*http.Response, error) {
+
+	resp, err := config.HTTPClient.Do(config.HTTPRequest)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
```

Comment:
Consider returning the error we get :)
```suggestion
	return resp, err
```